zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -141,6 +145,7 @@ pub(crate) struct ExecutionConfig<F> {
     caller_gadget: CallerGadget<F>,
     coinbase_gadget: CoinbaseGadget<F>,
     comparator_gadget: ComparatorGadget<F>,
+    copy_to_log_gadget: CopyToLogGadget<F>,
```

Comment:
move it to internal state gadgets part

Reply:
update in [755a06d](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/755a06d429eb994510af696e5c89d0c96f086176) thanks